### DISCUSSION TITLE
Why doesn't Projects detect my date fields?

### DISCUSSION THREAD

**Discussion by marcusolsson** [marcusolsson/obsidian-projects#309] [FAQ]
Projects only supports [ISO 8601](https://wikipedia.org/wiki/ISO_8601) date strings.

For example:

```md
---
date: 2006-01-28
---

# Weekly planning
```

**Comment by SimpleMedia1** [comment]
I had the same problem, and this is what I did:
Go to like a daily note where it is automatically generated or already working. and have the date: and copy and paste it, then in the projects views it should be generated and you will see a calendar icon under the date category.

**Comment by AzadKshitij** [comment]
If all your files don't have the date field in its metadata the plugin will not detect it as a date and show it as a text field.
